SCAutolib/models/file.py

```diff
@@ -315,6 +335,8 @@ def set(self, key: str, value: Union[int, str, bool], section: str = None):
         :param section: section of config file to be created/updated
         :type section: str
         """
+        self._changed = True
```

Comment:
What if I call `set` with the same value that is actually configured? Is it worth optimizing for this case?